Add 'forProperty' links from Procedure to Property

dr-shorthair opened the pull request:
... constrained to sub-class of Property where needed

Closes #47

dr-shorthair committed:
Add 'forProperty' links from Procedures, Systems, Deployment to Property, constrained to sub-class of Property where needed

dr-shorthair committed:
Fix diagrams

sgrellet commented on ssn/rdf/ontology/core/sosa-observation.ttl:
```diff
@@ -274,6 +274,8 @@ sosa:isPropertyOf
   rdfs:isDefinedBy sosa-common: .
 
 sosa:forProperty 
```
an owl:inverseOf would be very valuable for the communities ObsProp vocabularies and cwould then ompletely cover https://github.com/w3c/sdw-sosa-ssn/issues/47

dr-shorthair replied:
isPropertyOf inverseOf forProperty see aff1a93 - is this correct?

dr-shorthair replied:
Or should `isPropertyOf` be inverse of `hasProperty` ?? 
See https://w3c.github.io/sdw-sosa-ssn/ssn/#Features-of-Interest-and-Properties-overview

dr-shorthair replied:
I think I got this wrong. 
You want a link from a ObservableProperty (or ActuatableProperty) to the suitable Observation (and Actuation) Procedure(s). 
The inverse of `forProperty` _in this context_ could be `hasProcedure`?

dr-shorthair replied:
See [a852de4](https://github.com/w3c/sdw-sosa-ssn/pull/213/commits/a852de48844797d65482c8e5498a84ab0f492c55)

sgrellet replied:
sorry, was away for some weeks. 

`sosa:hasProcedure` was exactly what I was asking for. Thanks !

dr-shorthair replied:
@sgrellet Change your review to 'approve' ?

dr-shorthair committed:
Merge branch 'gh-pages' into 47-add-association-between-observingprocedure-and-observableproperty

dr-shorthair committed:
Inverse relation between forProperty and isPropertyOf

dr-shorthair committed:
Added `hasProcedure` to link from ObservableProperty or ActuatableProperty to Procedures that can observe/actuate the property
Updated figures to match
Changed Actuator->ActuatableProperty role to `actsOn` in parallel with `observes` for Observations

dr-shorthair committed:
Remove sosa:actsOn from this branch

dr-shorthair committed:
changelog

dr-shorthair commented:
Preview https://raw.githack.com/w3c/sdw-sosa-ssn/47-add-association-between-observingprocedure-and-observableproperty/ssn/index.html

dr-shorthair committed:
Merge branch 'gh-pages' into 47-add-association-between-observingprocedure-and-observableproperty

dr-shorthair committed:
Tidy up documentation and local range restrictions

dr-shorthair committed:
Note about implementation

dr-shorthair committed:
Merge branch 'gh-pages' into 47-add-association-between-observingprocedure-and-observableproperty

# Conflicts:
#	ssn/index.html

dr-shorthair committed:
Summary of proposed predicates - diagram

dr-shorthair commented:
Ping @KathiSchleidt @hylkevds @alexrobin - need one more review please
